test/specs/mainnet/swap/Smoke_CrossChainQuote.spec.js

```diff
@@ -0,0 +1,4195 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+
+import { assert } from "chai";
+import {
+  NETWORK_NAME_TO_CHAIN_ID,
+  CrossChainServiceProvider,
+  EnvNames,
+  NetworkNames,
+  Sdk,
+} from "etherspot";
+import { ethers } from "ethers";
+import pkg from "@etherspot/contracts";
+
+let sdkMainNet;
+let smartWalletAddress;
+let { ContractNames, getContractAbi } = pkg;
+
+describe("The SDK, when cross chain quote flow on the MainNet", () => {
+  // CROSS CHAIN QUOTES ON ARBITRUM NETWORK
+  it("Setup the SDK for Arbitrum network and perform the cross chain quote action", async () => {
+    // initialize the sdk
+    sdkMainNet = new Sdk(process.env.PRIVATE_KEY, {
+      env: EnvNames.MainNets,
+      networkName: NetworkNames.Arbitrum,
+    });
+
+    assert.strictEqual(
+      sdkMainNet.state.accountAddress,
+      "0xa5494Ed2eB09F37b4b0526a8e4789565c226C84f",
+      "The EOA Address is not displayed correctly."
+    );
+
+    // Compute the smart wallet address
+    let smartWalletOutput = await sdkMainNet.computeContractAccount();
+    smartWalletAddress = smartWalletOutput.address;
+    console.log("Smart wallet address: ", smartWalletAddress);
+
+    assert.strictEqual(
+      smartWalletAddress,
+      "0x666E17ad27fB620D7519477f3b33d809775d65Fe",
+      "The smart address is not displayed correctly."
+    );
+
+    let ArbitrumUSDC = "0xFF970A61A04b1cA14834A43f5dE4533eBDDB5CC8"; // Arbitrum - USDC
+    let MaticUSDC = "0x2791Bca1f2de4661ED88A30C99A7a9449Aa84174"; // Matic - USDC
+
+    let fromChainId = NETWORK_NAME_TO_CHAIN_ID[NetworkNames.Arbitrum];
+    let toChainId = NETWORK_NAME_TO_CHAIN_ID[NetworkNames.Matic];
+    let fromTokenAddress = ArbitrumUSDC;
+    let toTokenAddress = MaticUSDC;
+    let fromAmount = ethers.utils.parseUnits("0.5", 6);
+
+    let quoteRequestPayload = {
+      fromChainId: fromChainId,
+      toChainId: toChainId,
+      fromTokenAddress: fromTokenAddress,
+      toTokenAddress: toTokenAddress,
+      fromAmount: fromAmount,
+      serviceProvider: CrossChainServiceProvider.SocketV2, // Optional parameter
+    };
+    // console.log(quoteRequestPayload);
```

Comment:
Pleae remove any dead code

Reply:
Removed the commented code from the test cases.